package.json

```diff
@@ -40,6 +40,7 @@
     "@mantine/dates": "4.2.12",
     "@mantine/form": "4.2.12",
     "@mantine/hooks": "4.2.12",
+    "@mantine/modals": "^5.0.2",
```

Comment:
are there no interop issues between @mantine/modals v5+ and the other mantine dependencies?

Reply:
I don't think so, most of the breaking changes happened within the core package. I also checked out the code, and it worked fine.

Reply:
I originally tried simply upgrading Mantine to 5.0 across the board, which ended up breaking a LOT of stuff. I fixed it so it would compile and run again, but other unrelated things broke in strange ways, so the juice wasn't worth the squeeze.

I've tested pretty thoroughly, and things seem to work okay. If you feel strongly that I should revert "modals" to 4.2.12, I can, but I'll have to redo the confirm deletion popup (not too hard, thankfully).

Reply:
I think it's fine for now, because we want to migrate to Mantine v5 at some point anyway. I tracked the current progress here: https://github.com/LibrePhotos/librephotos/issues/583